src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -151,6 +166,10 @@ public Generator(GeneratorOptions o) {
   public Generator(GeneratorOptions o, Exporter.ExporterRuntimeOptions ero) {
     options = o;
     exporterRuntimeOptions = ero;
+    if (options.updatedPopulationSnapshotPath != null) {
+      exporterRuntimeOptions.deferExports = true;
```

Comment:
I saw later that "defer" just waits until the entire population is generated...

Reply:
If you serialize, it serializes the entire population all at once, and then exports them all at once.

Maybe we should think about serializing patients individually so we can reduce memory and speed up serialization. This might be future work.

Reply:
We could do that but bear in mind that the serialization includes the complete object graph reference from Person. You'd see a lot of duplication for each patient. I think to evolve a population we'd need to fire up an instance of synthea for each person, roll them forward and then quit.

Reply:
Why not have one instance of Synthea, using the thread pool, each thread load a single patient, roll them forward, serialize/export, and move to the next until all are processed?

Reply:
All the statics cause problems if you try to do that.